game/scripts/npc/abilities/dazzle_shallow_grave.txt

```diff
@@ -26,7 +26,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "60 45 30 23 23 22" //OAA
+    "AbilityCooldown"                                     "60 45 30 23 22 21" //OAA
```

Comment:
this is a bad buff as shallow grave with dazzle's ult (with aghs) is already really really good. I don't feel comfortable with this change as I feel dazzle to be in a good position as of now.

Reply:
Before Bad Juju, Dazzle could have perma-Grave with max level octarine. Now downtime is 1.5 seconds.

Reply:
I just don't see how you can justify buffing a hero thats's already extremely powerful. I would probably wait until after major to buff/nerf dazzle.

Reply:
If Dazzle is extremely powerful he would be a first ban or first pick in every scrim.

Reply:
ok timber isn't a first ban or first pick in every scrim let's buff him! That's not really the best logic to use. Remember only 3 heroes can be banned first phase, and some have priority over others. Remove heroes such as Axe and Dazzle is one of the greatest viable supports. There's also a difference between OP (PA) and Good (Dazzle). There's really no reason to buff him other than you want to at this point.